robomimic/utils/file_utils.py

```diff
@@ -260,6 +335,10 @@ def policy_from_checkpoint(device=None, ckpt_path=None, ckpt_dict=None, verbose=
     """
     ckpt_dict = maybe_dict_from_checkpoint(ckpt_path=ckpt_path, ckpt_dict=ckpt_dict)
 
+
+    env_meta = ckpt_dict["env_metadata"]
```

Comment:
@cremebrule Do we need these lines here (and the other print that I think you added for the env meta)? If not, we should probably remove them

Reply:
good point! We can remove them -- pretty sure it was for debugging.